samples/wifi/raw_tx_packet/src/main.c

```diff
@@ -167,6 +222,21 @@ static void fill_raw_tx_pkt_hdr(struct raw_tx_pkt_header *raw_tx_pkt)
 	raw_tx_pkt->queue = CONFIG_RAW_TX_PACKET_SAMPLE_QUEUE_NUM;
 	/* The byte is reserved and used by the driver */
 	raw_tx_pkt->raw_tx_flag = 0;
+
+	if (validate(CONFIG_RAW_TX_PACKET_SAMPLE_RATE_FLAGS, 0, 4, "Rate Flags")) {
```

Comment:
Why is this validation needed here? Why can `range` be used in the `Kconfig` itself ?

Reply:
This validation is just to ensure the error detection for inputs beyond the specified range in Kconfig.
Are there any concerns about defining both a range in Kconfig and implementing validation functions?

Reply:
Wouldn't it be redundant? If someone tries to specify a value which is out of range in the config then the compilation should not go through ?

Reply:
I wasn't aware of the usage of ``range``. After attempting to pass an out-of-range value, as you mentioned, it resulted in a compilation issue. Removing the functions. Thanks